content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,332 @@
+
+For the `name` field in subjects, and the `verbs`, `apiGroups`, `resources`,
+`namespaces`, and `nonResourceURLs` fields of resource and non-resource rules,
+the wildcard `*` may be specified to match all values for the given field,
+effectively removing it from consideration.
+
+A FlowSchema's `distinguisherMethod.type` determines how requests matching that
+schema will be separated into flows for the fair queuing algorithm. It may be
+either `ByUser`, in which case one requesting user will not be able to starve
+other users of capacity, or `ByNamespace`, in which case requests for resources
+in one namespace will not be able to starve requests for resources in other
+namespaces of capacity, or it may be blank (or `distinguisherMethod` may be
+omitted entirely), in which case all requests matched by this FlowSchema will be
+considered part of a single flow. The correct choice for a given FlowSchema
+depends on the resource and your particular environment.
+
+## Diagnostics
+Every HTTP response from an API server with the priority and fairness feature
+enabled will have two new headers, `X-Kubernetes-PF-FlowSchemaUID` and
+`X-Kubernetes-PF-PriorityLevelUID`, noting the flow schema that matched the request
+and the priority level to which it was assigned, respectively. The API objects'
+names are not included in these headers in case the requesting user does not
+have permission to view them, so when debugging you can use a command like 
+
+```shell
+kubectl get flowschemas -o custom-columns="uid:{metadata.uid},name:{metadata.name}"
+kubectl get prioritylevelconfigurations -o custom-columns="uid:{metadata.uid},name:{metadata.name}"
+```
+
+to get a mapping of UIDs to names for both FlowSchemas and
+PriorityLevelConfigurations.
+
+## Observability
+When you enable the API Priority and Fairness feature, the kube-apiserver
+exports additional metrics. Monitoring these can help you determine whether your
+configuration is inappropriately throttling important traffic, or find
+poorly-behaved workloads that may be harming system health.
+
+* `apiserver_flowcontrol_rejected_requests` counts requests that were rejected,
+  grouped by the name of the assigned priority level and the reason for
+  rejection.
+    * Reason may be `queue-full`, indicating that too many requests were already
+      queued (this includes the case where the PriorityLevelConfiguration is
+      configured to reject rather than queue excess requests), or `time-out`,
+      indicating that the request was still in the queue when its deadline
+      expired.
+
+* `apiserver_flowcontrol_current_inqueue_requests` gives the instantaneous total
+  number of queued (not executing) requests for each priority level.
+
+* `apiserver_flowcontrol_current_executing_requests` gives the instantaneous
+  total number of executing requests for each priority level.
+
+* `apiserver_flowcontrol_request_queue_length` gives a histogram of queue
+  lengths for the queues in each priority level.
```

Comment:
This is a histogram. Each contribution to the histogram reports the queue length seen, just after enqueuing, by a request that got enqueued. This is significantly different from an unbiased survey.